docs/Other SDL Features/Remote Control Vehicle Features/index.md

```diff
@@ -233,32 +384,164 @@ sdlManager.sendRPC(interiorVehicleData);
+
+Check the `allowed` property in the @![iOS]`SDLGetInteriorVehicleDataConsentResponse`!@@![android, javaSE, javaEE]`GetInteriorVehicleDataConsentResponse`!@ to see what modules can be controlled. Note that the order of the `allowed` array is 1-1 with the `moduleIds` array you passed into the @![iOS]`SDLGetInteriorVehicleDataConsent`!@@![android, javaSE, javaEE]`GetInteriorVehicleDataConsent`!@ RPC.
+
+!!! Note
+You should always try to get consent before setting any module data. If consent is not granted you should not attempt to set any module's data.
+!!!
+
+@![iOS]
+##### Objective-C
+```objc
+SDLGetInteriorVehicleDataConsent *getInteriorVehicleDataConsent = [[SDLGetInteriorVehicleDataConsent alloc] initWithModuleType:<#ModuleType#> moduleIds:@[@"<#ModuleID#>", @"<#ModuleID#>", @"<#ModuleID#>"]];
+[self.sdlManager sendRequest:getInteriorVehicleDataConsent withResponseHandler:^(__kindof SDLRPCRequest * _Nullable request, __kindof SDLRPCResponse * _Nullable response, NSError * _Nullable error) {
+    if (!response.success) { return; }
+    <#Allowed is an array of true or false values#>
+}];
+```
+##### Swift
+```swift
+let getInteriorVehicleDataConsent =  SDLGetInteriorVehicleDataConsent(moduleType: <#ModuleType#>, moduleIds: ["<#ModuleID#>", "<#ModuleID#>", "<#ModuleID#>"])
+sdlManager.send(request: getInteriorVehicleDataConsent , responseHandler: { (request, response, error) in
+    guard let res = response as? SDLGetInteriorVehicleDataConsentResponse else { return }
+    guard let allowed = res.allowed else { return }
+    let boolAllowed = allowed.map ({ (bool) -> Bool in
+        return bool.boolValue
+    })
+
+    <#boolAllowed is an array of true or false values#>
+})
+```
+!@
+
+@![android, javaEE, javaSE]
+```java
+// ToDo - Add example
+```
+!@
+
+#### Controlling a Module
+Below is an example of setting climate control data. It is likely that you will not need to set all the data as in the code example below. When connected to RPC 6.0+ systems, you must set the `moduleId` in @![iOS]`SDLSetInteriorVehicleData.moduleData`!@@![android, javaSE, javaEE]`SetInteriorVehicleData.setModuleData`!@. When connected to < 6.0 systems, there is only one module per module type, so you must only pass the type of the module you wish to control.
+
+When you received module information above in **Getting Remote Control Module Information** on RPC v6.0+ systems, you received information on the `location` and `serviceArea` of the module. The permission area of a module depends on that `serviceArea`. The `location` of a module is like the `seats` array: it maps to the `grid` to tell you the physical location of a particular module. The `serviceArea` maps to the grid to show how far that module's scope reaches.
```

Comment:
Looks like we can create anchor links to headers (https://github.com/smartdevicelink/sdl_markdown_spec#header-anchor) 

The **Getting Remote Control Module Information** section can be linked like so: 
[Getting Remote Control Module Information](#-getting-remote-control-module-information)

Reply:
🆗